config/kubernetes-sigs/sig-scheduling/teams.yaml

```diff
@@ -12,20 +12,25 @@ teams:
   descheduler-admins:
     description: Admin access to the descheduler repo
     members:
-    - aveshagarwal
     - damemi
-    - k82cn
-    - ravisantoshgudimetla
+    - ingvagabund
+    # Emeritus:
+    # - aveshagarwal
+    # - k82cn
+    # - ravisantoshgudimetla
     privacy: closed
   descheduler-maintainers:
     description: Write access to the descheduler repo
     members:
-    - aveshagarwal
+    - a7i
     - damemi
     - ingvagabund
-    - k82cn
-    - ravisantoshgudimetla
+    - knelasevero
```

Comment:
why so many contributors with write access, as opposed to just approval rights?

Reply:
Hey sorry for the delay on this. Our goal is to have more active maintainers who have access to publish releases (mainly pushing git tags to the repo). I copied this list from the current list of approvers in the repo. But we could probably shorten it if this is too broad

Reply:
write access could be dangerous, so I would be cautious about who has the permissions. Ultimately it's your decision as TL for descheduler, but I wanted to make sure you considered this.

Reply:
That's a valid point. I updated this change to only add @a7i as a writer (he is actively working on publishing this release).

We have an action item to automate our release process, so we will follow up with this to reduce the number of people with direct write access.